web-app/src/main/webapp/script.js

```diff
@@ -12,23 +12,25 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+let map;
```

Comment:
nit: I think this file is going to get big soon. I think it's worth doing a refactor PR after this one to refactor this into a class or something. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes

Reply:
I made a refactor PR that removes use of global variable instead!